app/src/main/java/com/hapramp/ui/fragments/EarningFragment.java

```diff
@@ -263,6 +413,68 @@ public void onUserSavingSBD(final String savingSBD) {
     }
   }
 
+  @Override
+  public void onUserRewards(String _sbdReward, String _steemReward, String _rewardVests) {
+    StringBuilder rewardPanelText = new StringBuilder();
+    boolean rewardExists = false;
+    double sbdReward = Double.parseDouble(_sbdReward.split(" ")[0]);
+    double steemReward = Double.parseDouble(_steemReward.split(" ")[0]);
+    double vestsReward = Double.parseDouble(_rewardVests.split(" ")[0]);
+    finalSBDReward = _sbdReward;
+    finalSteemReward = _steemReward;
+    finalVestsReward = _rewardVests;
+    try {
+      if (sbdReward > 0) {
+        rewardPanelText.append(finalSBDReward);
+        rewardExists = true;
+      }
+      if (steemReward > 0) {
+        rewardPanelText.append(finalSteemReward);
+        rewardExists = true;
+      }
+      if (vestsReward > 0) {
+        rewardPanelText.append(String.format(Locale.US, "%.4f SP",
```

Comment:
Will it be 4 digit decimal precision? I think we should stick to 3.

Reply:
Ok!